explorer/lib/explorer_web/components/labeled_progress_bar.ex

```diff
@@ -0,0 +1,32 @@
+defmodule LabeledProgressBarComponent do
+  use ExplorerWeb, :live_component
+
+  attr :progress, :float, required: true
+  attr :label, :string, required: true
+
+  @impl true
+  def render(assigns) do
+  ~H"""
+  <div class="progress-with-labels">
+    <div
+    class="progress-foreground"
+    style={"clip-path: inset(0 #{100 - @progress}% 0 0);"}
+    >
+    <%= @label %>
+    </div>
+    <div
+    class="progress-background"
+    style={"clip-path: inset(0 0 0 #{@progress}%);"}
+    >
+    <%= @label %>
```

Comment:
We can remove this label right?

Reply:
I don't think so, that label is the text we see and one hides the other AFAICT.